tests/dm_syncer/data/db1.prepare.sql

```diff
@@ -3,3 +3,39 @@ create database `dm_syncer`;
 use `dm_syncer`;
 create table t1 (id int, name varchar(20), primary key(`id`));
 insert into t1 (id, name) values (1, 'arya'), (2, 'catelyn');
+
+drop database if exists `logs`;
```

Comment:
we could rename these database to "dm_syncer_do_db", "dm_syncer_ignore_db" or something, to avoid other test use same name